data/coalition/lunarium intro.txt

```diff
@@ -632,7 +632,7 @@ mission "House Bebliss 2-FW"
 				`	"Human space was invaded by a species known as the Pug."`
 			label pug
 			`	Thankfully, the ones that were downing drinks weren't that close when you said the word "Pug," or else you'd need a new set of clothes. One of them signals for a maid to clean up their simultaneous spit takes, and the Arach by the blueprints finally comes closer.`
-			`	"'Pug,' were they called?" she asks as the maid behind her quickly scrubs the floor. You affirm her question, and she calls for a servant, asking them to ready a ship. She whispers something to the other two members of House Bebliss, which the translation devices don't pick up, and starts gathering her belongings. "Forgive me, Captain, but leave at once, I must."`
+			`	"'Pug,' were they called?" she asks as the maid behind her quickly scrubs the floor. You affirm her question, and she calls for a servant, asking them to ready a ship. She whispers something to the other two members of House Bebliss, which the translation devices don't pick up, and starts gathering her belongings. "To forgive me, I ask you, Captain, but leave at once, I must."`
```

Comment:
```suggestion
			`	"'Pug,' were they called?" she asks as the maid behind her quickly scrubs the floor. You answer in the affirmative, and she calls for a servant, asking them to ready a ship. She whispers something to the other two members of House Bebliss, which the translation devices don't pick up, and starts gathering her belongings. "To forgive me, I ask you, Captain, but leave at once, I must."`
```

Reply:
I agree, it does sound kind of verbose. On the other hand, "affirming" someone question doesn't feel like correct English. What do you think?